lib/components/CircuitJsonPreview/CircuitJsonPreview.tsx

```diff
@@ -104,21 +129,44 @@ export const CircuitJsonPreview = ({
   }
 
   useEffect(() => {
-    if (errorMessage) {
+    if (circuitJsonErrors || errorMessage) {
```

Comment:
In my last review, i mentioned we do not want to switch the tab in the case of circuit json errors